Client/Clique/Common/Services/Session.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Session.swift
+//  Clique
+//
+//  Created by Infinum on 22.12.2022..
+//
+
+import Foundation
+import JWTDecode
+
+final class Session {
+    func getUserId() -> Int {
+        var id = 0
+        do {
+            let jwt = try decode (jwt: UserStorage.token!)
+            if let userId = jwt["UserId"].string {
+                id = Int(userId) ?? 0
+            }
+        } catch {
+            return 0
```

Comment:
I'm not sure if this is valid - is the `0` a valid `userId`? If this is returned, will it work properly?
If not, rather return `Int?`, and when the value is not there, just return `nil`.

Reply:
There's no user with id 0, that's why I wrote it here like that.

Reply:
Okay, I get that point, but I was thinking from a different perspective.
E.g., if there is `userId`, I would expect that session is active and there is a user currently in this session.

Still, now you should know how this method works internally and that `0` represents that there is no valid `userId`. This means you need to know how the backend works (`0` will not be returned as a valid id).

If that value was an `Optional` like `Int?`, you would know if that value is `nil`, there is no valid session (or user in the session). Otherwise, if the value is returned, you would know the returned value is a valid number (no need to know if some value can be defined on the backend side or if specific id means that the value is not a valid id).

It can stay like this, but I'm just giving you an example how to remove additional logic may complicate future implementation (e.g. someone else may not know this is the case).